### ISSUE TITLE
Operator "+" should work

### ISSUE THREAD

**Issue by RobinHankin** [RobinHankin/stokes#1]
kforms are linear, so I would expect numerical verification of linearity to be easy.  The output of the last line below shows that this is not the case:

```
 k1 <- kform_general(4,2,rnorm(6))
 k2 <- kform_general(4,2,rnorm(6))
 k3 <- k1+k2
 E <- matrix(rnorm(8),4,2)
 as.function(k1)(E)
[1] -7.553959
 as.function(k2)(E)
[1] 2.911626
 as.function(k3)(E)
[1] -2.7909217  0.4114198 -1.9531770  0.2841144
```

I would expect the last line to be ```as.function(k1)(E) + as.function(k2)(E)``` but  because ```k3``` is not a kform object, ```as.function(k3)``` is dispatches to ```as.function.spray()```

**Comment by RobinHankin** [comment]
also observe that 

```as.function(2*k1)(E)```

does not work.

**Comment by RobinHankin** [comment]
fixed in 916cca8effa37f5baad5a1147b4e974364b7ab51
